services/r2ctl/main/main.go

```diff
@@ -29,15 +29,20 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/m3db/m3ctl/health"
+	"github.com/m3db/m3ctl/r2"
+	"github.com/m3db/m3ctl/r2/kv"
+	"github.com/m3db/m3ctl/services/r2ctl/server"
+	"github.com/m3db/m3ctl/swagger"
+
```

Comment:
nit: skip this new line or make the other files follow this convention too (e.g. `r2/r2.go` groups `m3db/...` org and package imports)